submarine-server/server-submitter/submitter-k8s/src/main/java/org/apache/submarine/server/submitter/k8s/K8sSubmitter.java

```diff
@@ -539,9 +537,9 @@ public void createServe(ServeSpec spec)
       api.createNamespacedCustomObject(istioVirtualService.getGroup(),
               istioVirtualService.getVersion(),
               "default",
-              istioVirtualService.getPlural(),
-              istioVirtualService,
-              "true");
```

Comment:
This is a mistake of replace istioVirtualService with seldonDeployment

Reply:
Oh, you're right. I'll fix it ASAP.